src/community.rs

```diff
@@ -2,26 +2,34 @@ use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{env, AccountId};
 
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Wiki {
+    name: String,
+    content_markdown: String,
+}
+
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
 #[serde(crate = "near_sdk::serde")]
 pub struct Community {
     pub name: String,
     pub description: String,
     pub image_url: String,
     pub thumbnail_url: String,
-    pub overview_page_markdown: String,
-    pub events_page_markdown: String,
+    pub wiki1: Option<Wiki>,
+    pub wiki2: Option<Wiki>,
     pub admins: Vec<AccountId>,
-    pub labels: Vec<String>,
-    pub telegram_handles: Vec<String>,
+    pub tag: String,
```

Comment:
How `Community::tag` is different from `CommunityCard::handle`? Is it the same thing? It is confusing, please, either rename it to be the same or add comments in the code 🙏

Reply:
@frol – I think we were thinking of renaming "labels" to "tags" to make it consistent with NEAR Social conventions. And handle would be the URL slug.

Example:

- Zero Knowledge Tag: `zk`
- Zero Knowledge Handle: `zero-knowledge` e.g. /community/zero-knowledge 

Please provide your input on how to make this less confusing. Perhaps we need to rename handle to URL slug. Or maybe just keep labels for now.

Reply:
@ori-near Why do we need those values to be different? If you want to use `zero-knowledge` in the URL, let's just apply `zero-knowledge` label to the posts where it is missing (either replace `zk` with it, or just add another label to the posts), and use `zero-knowledge` to filter the posts.

Reply:
@frol handles can be converted into account names later. Or is it okay to use accountId as a tag as well? 🤔